### ISSUE TITLE
NoSuchMethodError: The method 'cancel' was called on null at AnsiSpinner.finish

### ISSUE THREAD

**Issue by jmagman** [flutter/flutter#75677]
beta crasher:
```
NoSuchMethodError: The method 'cancel' was called on null.
Receiver: null
Tried calling: cancel()
  | at Object.noSuchMethod | (object_patch.dart:54)
  | at AnsiSpinner.finish | (logger.dart:952)
  | at Status.stop | (logger.dart:775)
  | at AnsiStatus.stop | (logger.dart:1030)
  | at DevtoolsServerLauncher._activateDevTools | (devtools_launcher.dart:167)
  | at <asynchronous gap> | (async)
  | at DevtoolsServerLauncher.launch | (devtools_launcher.dart:71)
  | at <asynchronous gap> | (async)
  | at DevtoolsServerLauncher.serve | (devtools_launcher.dart:176)
  | at <asynchronous gap> | (async)
...
```
https://crash.corp.google.com/browse?q=product_name%3D%27Flutter_Tools%27+AND+product.Version%3D%27beta%2F1.26.0-17.3.pre%27+AND+stable_signature%3D%27NoSuchMethodError+at+Object.noSuchMethod+at+dart%3Acore-patch%2Fobject_patch.dart-44ba5769%27

**Comment by jmagman** [comment]
This is the #6 highest crasher in the tool on 2.0.1, request for hotfix.

**Comment by findtravishere** [comment]
I'm still receiving this error on 2.0.2 latest stable release as of today

**Comment by jmagman** [comment]
@findtravishere The fix https://github.com/flutter/flutter/pull/75679 is available in [1.27.0-4.0.pre](https://flutter.dev/docs/development/tools/sdk/releases?tab=macos), which is currently available on the [dev channel](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels#how-to-change-channels).  I've added a request to hotfix this to stable but it hasn't happened yet.
